BootloaderCommonPkg/Include/Library/IasImageLib.h

```diff
@@ -69,14 +69,32 @@ typedef struct {        /* a file (sub-image) inside a boot image */
 } IMAGE_DATA;
 
 //
-// IAS Pub Key format for use with IPP CryptoLib
+// RSA Pub Key format for use with IPP CryptoLib
 //
 typedef struct {
-  UINT32  Signature;
-  UINT8 PubKeyMod[RSA_MOD_SIZE];
-  UINT8 PubKeyExp[RSA_E_SIZE];
+  UINT32                PubExp;
+  UINT8                 Modulus[256];
 } IAS_PUB_KEY;
 
+//
```

Comment:
Is there any benefit in re-defining these data structs here?
Can't we just add dependency of CryptoLib on IasImageLib and use the data structs from there?

Reply:
since we will not add 3K support for old IAS image format. Not change anything on IAS if possible.

Reply:
Yeah, we can add dependency on CryptoLib and re-use the same data structures.